server/handler.go

```diff
@@ -62,15 +61,6 @@ const (
 	MultiStmtModeOn  MultiStmtMode = 1
 )
 
-func init() {
-	// Set the log.Error and log.Errorf functions in Vitess so that any errors
-	// logged by Vitess will appear in our logs. Without this, errors from Vitess
-	// can be swallowed (e.g. any parse error for a ComPrepare event is silently
-	// swallowed without this wired up), which makes debugging failures harder.
-	log.Error = logrus.StandardLogger().Error
-	log.Errorf = logrus.StandardLogger().Errorf
-}
-
```

Comment:
Turns out `log.go` does correctly wire up the Vitess loggers to the GMS loggers. I tested and confirmed that it was working correctly without this. I believe when we added this, it was also added with some additional logging output, and I think those were the real logging gap. This doesn't seem to be needed, so cleaning up.